index.js

```diff
@@ -178,9 +199,20 @@ function getCarInfoById(/* code here */) {
  *     (1) an array which is an inventory of cars like the one inside /data/inventory.js.
  * sortCarInventory returns an inventory that is sorted by car_model, ascending [A-Z].
 */
-function sortCarInventory(/* code here */) {
-  /* code here */
-}
+function sortCarInventory(inventory) {
+  return inventory.sort(function (model1,model2){
+    let a = model1.car_model.toUpperCase();
+    let b = model2.car_model.toUpperCase();
```

Comment:
Not sure what the purpose of toUpperCase(); is here, it was not asking for that in the instructions.